badgerdb/badgerdb.go

```diff
@@ -97,6 +98,38 @@ type Options struct {
 	// Directory for storing the DB files.
 	// Optional ("BadgerDB" by default).
 	Dir string
+	// Uses in-memory storage, ignores file-specified options.
+	// Optional (false by default).
+	InMemory bool
+	// When SyncWrites is true all writes are synced to disk.
+	// Optional (false by default).
+	SyncWrites bool
+	// When ReadOnly is true the DB will be opened on read-only mode.
+	// Multiple processes can open the same Badger DB.
+	// Optional (false by default).
+	ReadOnly bool
+	// Truncate indicates whether value log files should be truncated to delete corrupt data, if any.
+	// This option is ignored when ReadOnly is true.
```

Comment:
Do we have to take care of ignoring this option, or can we just pass `ReadOnly` and `Truncate` to the BadgerDB options and it takes care of this? I guess this is the case, just want to make sure here.